### DISCUSSION TITLE
1.2.6 download link

### DISCUSSION THREAD

**Discussion by actuallyitsellie** [TitanNano/TS4ControlAnySim#101] [Q&A] [answered]
I can't find the download link for version 1.2.6 (the one that fixed it for macs)

**Comment by TitanNano** [comment, marked as answer]
The download link is here: https://github.com/TitanNano/TS4ControlAnySim/releases/tag/v1.2.6 you just download the first zip file under Assets. The update is coming to CurseForge and MTS, but CurseForge is being really slow with reviewing and approving the new version....

**Reply by TitanNano** [reply]
Update is now also available on curseforge.
